### PR TITLE
Edit readme

### PR DESCRIPTION
**Pull request by cconrad8**
I wanted to provide some more details on the documentation so I/others in the future don't forget.
Please add any thoughts!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,49 @@
...
 * `Attribute` : (REQUIRED) Name of the new concept.  
 * `Description` : (REQUIRED) A description for the concept.  
-* `Valid Values` : Only required if you are editing an `annotationProperty.csv` file, where you would be defining a new field (relation) and specifying what values are allowed to be used with this field (relation).  
-For example, _tumorType_ is to valid values _{ Malignant Peripheral Sheath Tumor, Schwannoma, ... }_ as _eats_ is to valid values _{ pizza, sandwich, egg, ... }_.    
+* `Valid Values` : Only required if you are editing an `annotationProperty.csv` file, where you would be defining a new field (relation) and specifying what values are allowed to be used with this field (relation). For example, _tumorType_ is to valid values _{ Malignant Peripheral Sheath Tumor, Schwannoma, ... }_ as _eats_ is to valid values _{ pizza, sandwich, egg, ... }_.    
 * `Source` : Preferably a URI to an ontology source term.
+If you need help, you can do a pull request and the maintainer will help fill out the rest.  
+4. If you are adding an attribute that is a valid value for another attribute, navigate to that csv file and add the term.
```
Comment (anngvu) [inline README.md:19]: Duplicate step 4

File: README.md [hunk 2]
```diff
@@ -1,56 +1,49 @@
...
+5. Create a pull request (PR) and add @allaway or @anngvu as a reviewer. Creating the PR will perform an action to update the [NF.csv](https://github.com/nf-osi/nf-metadata-dictionary/blob/main/NF.csv) file and the [NF.jsonld](https://github.com/nf-osi/nf-metadata-dictionary/blob/main/NF.jsonld) from the modules folder. **You do not need to edit the `NF.csv` or `NF.jsonld` directly because it will be done automatically**
```
Comment (anngvu) [inline README.md:20]: Why not `add @allaway, @anngvu, or @cconrad8 as a reviewer`. You are a maintainer now! 😁

**Review by anngvu — CHANGES_REQUESTED** [review]
Hey, this is a good reorg! Especially agree moving the versioning stuff to where they make more sense. Made some suggestions. I do think it might be good to say something like "Step 1 -6 anyone can do, but merging and releases can only be done by a maintainer".

File: README.md (hunk 2 above)
Comment (cconrad8) [reply README.md]: @anngvu thanks for the suggestions! I try to change it so it applies to both the contributor and us (maintainer). 
Let me know your thoughts!

**Review by anngvu — APPROVED** [review]
This looks awesome! 👍🏼 👍🏼

**Pull request merged**